### PR TITLE
docs(cpp): fix typo in documentation

### PR DESCRIPTION
**Pull request by vonhyou**
## 📑 Description

Fix a typo in `pack/cpp/README.md`

### CODE REVIEW

File: lua/astrocommunity/pack/cpp/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This plugin pack does the following:
 - Adds [clangd_extensions.nvim](https://github.com/p00f/clangd_extensions.nvim) for language specific tooling
 - Adds [cmake-tools.nvim](https://github.com/Civitasv/cmake-tools.nvim) for building and debugging
 
-tips: the offset_encondings of clangd will confilicts whit null-ls, so add the following file in your user_config.
```
Comment (Uzaaft) [inline lua/astrocommunity/pack/cpp/README.md:11]: confilicts is also wrong. Could you fix that typo while you're at it? :)

Comment (vonhyou) [reply lua/astrocommunity/pack/cpp/README.md:11]: Hi, I've fixed it. Thank you.

Comment (Uzaaft) [reply lua/astrocommunity/pack/cpp/README.md:11]: No it's still wrong.

Comment (vonhyou) [reply lua/astrocommunity/pack/cpp/README.md:11]: Please check [b79b368e82dcadca69bed7d983478ddcbbc38bf5](https://github.com/AstroNvim/astrocommunity/pull/594/commits/b79b368e82dcadca69bed7d983478ddcbbc38bf5) and the latest commit

File: lua/astrocommunity/pack/cpp/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This plugin pack does the following:
...
+tips: the offset_encondings of clangd will confilict with null-ls, so add the following file in your user_config.
```
Comment (Uzaaft) [inline lua/astrocommunity/pack/cpp/README.md:11]:
```suggestion
tips: the offset_encondings of clangd will conflict with null-ls, so add the following file in your user_config.
```

**Review by Uzaaft — CHANGES_REQUESTED** [review]

File: lua/astrocommunity/pack/cpp/README.md (hunk 1 above)
Comment (Uzaaft) [reply lua/astrocommunity/pack/cpp/README.md:11]: I saw those. It's still wrong. It should be conflict. Next time please double check :)

**Review by Uzaaft — APPROVED** [review]

**Pull request merged**